demo/admin/src/App.tsx

```diff
@@ -52,132 +46,120 @@ const GlobalStyle = () => (
+                additionalToolbarItems: <ImportFromPicsum />,
+                importSources: {
+                    picsum: {
+                        label: <FormattedMessage id="dam.importSource.picsum.label" defaultMessage="Lorem Picsum" />,
+                    },
+                },
+                contentGeneration: {
+                    generateAltText: true,
+                    generateImageTitle: true,
+                },
+            }}
+            imgproxy={config.imgproxy}
+            dependencies={{
+                entityDependencyMap: {
+                    Page,
+                    Link,
+                    News: NewsDependency,
+                    DamFile: createDamFileDependency(),
+                },
+            }}
+            siteConfigs={{
+                configs: config.siteConfigs,
+                resolveSiteConfigForScope: (configs, scope) => {
+                    const siteConfig = configs.find((config) => {
+                        return config.scope.domain === scope.domain;
+                    });
 
-                                if (!siteConfig) throw new Error(`siteConfig not found for domain ${scope.domain}`);
-                                return {
-                                    url: siteConfig.url,
-                                    preloginEnabled: siteConfig.preloginEnabled || false,
-                                    blockPreviewBaseUrl:
-                                        siteConfig.scope.domain === "secondary"
-                                            ? `${siteConfig.url}/block-preview`
-                                            : `${siteConfig.url}/block-preview/${scope.domain}/${scope.language}`,
-                                    sitePreviewApiUrl: `${siteConfig.url}/site-preview`,
-                                };
-                            },
+                    if (!siteConfig) throw new Error(`siteConfig not found for domain ${scope.domain}`);
+                    return {
+                        url: siteConfig.url,
+                        preloginEnabled: siteConfig.preloginEnabled || false,
+                        blockPreviewBaseUrl:
+                            siteConfig.scope.domain === "secondary"
+                                ? `${siteConfig.url}/block-preview`
+                                : `${siteConfig.url}/block-preview/${scope.domain}/${scope.language}`,
+                        sitePreviewApiUrl: `${siteConfig.url}/site-preview`,
+                    };
+                },
+            }}
+            buildInformation={{ date: config.buildDate, number: config.buildNumber, commitHash: config.commitSha }}
+            contentLanguage={{ resolveContentLanguageForScope: (scope: ContentScope) => scope.language }}
+            blocks={{ context: { demo: true } }}
+        >
+            <ApolloProvider client={apolloClient}>
+                <IntlProvider locale="en" messages={getMessages()}>
+                    <BlocksConfigProvider
```

Comment:
didn't you want to add this also into CometConfigProvider? (for default config - with possibility to add nested BlocksConfigProvider)

Reply:
Added: dc28d23326091cf45ce5603d72ef8df090dd6278